email-ballerina/Ballerina.toml

```diff
@@ -0,0 +1,39 @@
+[project]
+org-name = "ballerina"
+version = "0.7.0"
```

Comment:
Shall we parameterize this version like
https://github.com/ballerina-platform/module-ballerina-grpc/blob/master/grpc-ballerina/Ballerina.toml#L3

Reply:
Parameterized.